hydra-http.c

```diff
@@ -426,37 +446,70 @@ int32_t service_http_init(char *ip, int32_t sp, unsigned char options, char *mis
   //   -1  error, hydra will exit, so print a good error message here
 
   /*POU CODE */
-  char *start = strstr(miscptr, "F=");
-  if (start == NULL)
-    start = strstr(miscptr, "S=");
-
-  if (start != NULL) {
-    if (start[0] == 'F')
-      end_condition_type = 0;
-    else
-      end_condition_type = 1;
-
-    int condition_len = strlen(start);
-    memset(end_condition, 0, END_CONDITION_MAX_LEN);
-    if (condition_len >= END_CONDITION_MAX_LEN) {
-      hydra_report(stderr, "Condition string cannot be bigger than %u.", END_CONDITION_MAX_LEN);
-      return -1;
+
+  char *delim = ":";
+  strtok(miscptr, delim);
+  char *p;
+
+  while ((p = strtok(NULL, delim))) {
+    if (strstr(p, "F=") != NULL || strstr(p, "S=") != NULL) {
```

Comment:
you are changing the behaviour here.
the S=/F= is optional, is no (char)=  is given it defaults to F=
also note for F=/S=:`Note: this must be the last option supplied.`
But I would not mind to change this, that always F=/S= has to be given and that it doesnt need to be the last parameter. But then a) the documentation has to be updated (help output) and b) the http-form module updated as well.